# fix internal compiler error for {intel,iomkl}/2018b (icc v2018.3.222/18.0.3)
# August 21st 2018 by B. Hajgato (Free Univeristy Brussels - VUB)
--- scipy-0.10.1/scipy/spatial/qhull.c.orig	2011-12-29 11:08:35.000000000 +0100
+++ scipy-0.10.1/scipy/spatial/qhull.c	2018-08-20 11:21:37.066897557 +0200
@@ -894,6 +894,7 @@
 static int __pyx_f_5scipy_7spatial_5qhull__barycentric_inside(int, double *, double *, double *, double); 
 static void __pyx_f_5scipy_7spatial_5qhull__barycentric_coordinate_single(int, double *, double *, double *, int); 
 static void __pyx_f_5scipy_7spatial_5qhull__barycentric_coordinates(int, double *, double *, double *); 
+#pragma intel optimization_level 1
 static void __pyx_f_5scipy_7spatial_5qhull__lift_point(__pyx_t_5scipy_7spatial_5qhull_DelaunayInfo_t *, double *, double *); 
 static double __pyx_f_5scipy_7spatial_5qhull__distplane(__pyx_t_5scipy_7spatial_5qhull_DelaunayInfo_t *, int, double *); 
 static int __pyx_f_5scipy_7spatial_5qhull__is_point_fully_outside(__pyx_t_5scipy_7spatial_5qhull_DelaunayInfo_t *, double *, double); 
